.bazelrc

```diff
@@ -67,6 +67,7 @@ build:cuda12_2 --define=use_cuda12=true
 build:cuda12_2 --copt="-DUSE_CUDA12=1"
 build:cuda12_2 --copt="-DUSE_OLD_TRT_FMHA=1"
 
+build:cpu --copt="-DENABLE_BF16=1"
```

Comment:
why change behaviour of cpu compile?

Reply:
I've removed ENABLE_BF16=1 from the "base" config and moved it to cuda. I've added it to cpu one because it used to get that from the base config. We disabled ENABLE_BF16 for time being on rocm.